dev/com.ibm.ws.org.apache.commons.io/bnd.overrides

```diff
@@ -7,13 +7,13 @@ Import-Package: !org.apache.commons.io.*, *
 
 Export-Package: \
 org.apache.commons.io;version="1.4.9999",org.apache.commons.io.comparator;version="1.4.9999", \
-org.apache.commons.io.comparator;version="2.8.0",org.apache.commons.io.file;version="2.8.0", \
-org.apache.commons.io.filefilter;version="1.4.9999",org.apache.commons.io.filefilter;version="2.8.0", \
-org.apache.commons.io.function;version="2.8.0",org.apache.commons.io.input;version="1.4.9999", \
-org.apache.commons.io.input.buffer;version="2.8.0",org.apache.commons.io.input;version="2.8.0", \
-org.apache.commons.io.monitor;version="2.8.0",org.apache.commons.io.output;version="1.4.9999", \
-org.apache.commons.io.output;version="2.8.0",org.apache.commons.io.serialization;version="2.8.0", \
-org.apache.commons.io;version="2.8.0"
+org.apache.commons.io.comparator;version="2.11.0",org.apache.commons.io.file;version="2.11.0", \
+org.apache.commons.io.filefilter;version="1.4.9999",org.apache.commons.io.filefilter;version="2.11.0", \
+org.apache.commons.io.function;version="2.11.0",org.apache.commons.io.input;version="1.4.9999", \
+org.apache.commons.io.input.buffer;version="2.11.0",org.apache.commons.io.input;version="2.11.0", \
+org.apache.commons.io.monitor;version="2.11.0",org.apache.commons.io.output;version="1.4.9999", \
+org.apache.commons.io.output;version="2.11.0",org.apache.commons.io.serialization;version="2.11.0", \
+org.apache.commons.io.file.spi;version="2.11.0",org.apache.commons.io;version="2.11.0"
```

Comment:
org.apache.commons.io package is the first one listed.  No reason to add it here a second time.  Just update the version appropriately above.

Reply:
Not sure why org.apache.commons.io.comparator and org.apache.commons.io.filefilter is also duplicated.